batchglm/train/numpy/base_glm/estimator.py

```diff
@@ -86,8 +94,95 @@ def train_sequence(self, training_strategy, **kwargs):
                         "overrding %s from training strategy with value %s with new value %s\n"
                         % (x, str(d[x]), str(kwargs[x]))
                     )
+            if "dispersion_smoothing" in d:
```

Comment:
`"dispersion_smoothing" in d` a variable since is re-used

Reply:
Not sure what you meant but it seems you don't like the way I first checked for the existence and then for the value of the dispersion_smoothing method and that multiple times for different flavours. I agree and replaced this with a simple `ds_method = d.get('dispersion_smoothing', 'no_smoothing')` so it uses `no_smoothing` as a default value if not found.